thirdeye-ui/src/app/components/anomalies-create/create-anomaly-properties-form/create-anomaly-properties-form.component.tsx

```diff
@@ -0,0 +1,249 @@
+
+export const CreateAnomalyPropertiesForm: FunctionComponent<CreateAnomalyPropertiesFormProps> =
+    ({
+        alerts,
+        formFields,
+        readOnlyFormFields,
+        handleSetField,
+        enumerationItemsForAlert,
+        enumerationItemsStatus,
+        timezone,
+    }) => {
+        const { t } = useTranslation();
+
+        const showEnumerationItemsField = !!(
+            formFields.alert &&
+            getEnumerationItemsConfigFromAlert(formFields.alert)
+        );
+
+        const formLabels: Record<CreateAnomalyFormKeys, string> = {
+            alert: t("label.alert"),
+            dataSource: t("label.datasource"),
+            dataset: t("label.dataset"),
+            dateRange: t("label.date-range"),
+            enumerationItem: t("label.dimension"),
+            metric: t("label.metric"),
+        };
+
+        const timeRangeDuration = createTimeRangeDuration(
+            TimeRange.CUSTOM,
+            formFields.dateRange[0],
+            formFields.dateRange[1]
+        );
+
+        return (
+            <>
+                <InputSection
+                    inputComponent={
+                        <Autocomplete
+                            disableClearable
+                            fullWidth
+                            getOptionLabel={(option) => option.name}
+                            options={alerts}
+                            renderInput={(params) => (
+                                <TextField
+                                    {...params}
+                                    InputProps={{
+                                        ...params.InputProps,
+                                    }}
+                                    margin="dense"
+                                    placeholder={capitalize(
+                                        t(
+                                            "message.click-here-to-select-entity",
+                                            {
+                                                entity: t("label.alert"),
+                                            }
+                                        )
+                                    )}
+                                    size="small"
+                                    variant="outlined"
+                                />
+                            )}
+                            value={formFields.alert || undefined}
+                            onChange={(_, selectedValue) => {
+                                selectedValue &&
+                                    handleSetField("alert", selectedValue);
+                            }}
+                        />
+                    }
+                    labelComponent={
+                        <>
+                            <Typography variant="body2">
+                                {formLabels.alert}
+                            </Typography>
+                            {!!formFields.alert &&
```

Comment:
I don't think we need `!!` when doing these but let me know if you encountered an issue

Reply:
Same reason as [the comment above](https://github.com/startreedata/thirdeye/pull/1009#discussion_r1130460398), to coerce the left hand side into being `boolean`